mesh/mesh.cpp

```diff
@@ -2555,82 +2556,110 @@ void Mesh::ReorderElements(const Array<int> &ordering, bool reorder_vertices)
 }
 
 
+void Mesh::GetEdgeLengths(const DSTable &v_to_v, Array<double> &lengths) const
```

Comment:
Should we also change these names to `GetEdgeLengths2` or `GetEdgeLengthsSquared` and `lengths2` or `sqlengths`?

Reply:
See [6347978](https://github.com/mfem/mfem/pull/3847/commits/6347978eb2b59f1056317f86a1edc197ebf1e7bf). I think renaming the function is a good idea but keeping the variable name as `lengths` is fine.